lib/topology_docker_openswitch/openswitch.py

```diff
@@ -221,6 +280,9 @@ def __init__(
             **kwargs
         )
 
+        # Save location of the shared dir in host
+        self.shared_dir = shared_dir
+
```

Comment:
Oh great! As a matter of fact I was reusing that part of the code from Pablo's code, so no worries I'll delete it right away. Thanks Carlos!